### PR TITLE
[README] updated CUDA instructions in README

### PR DESCRIPTION
**Pull request by tijyojwad**
1. Updated README for correct CUDA dependencies and instructions
2. Moved our custom end to end test scripts to `ci` folder to cleanly separate them from racon tests

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -138,9 +108,15 @@ Usage of `racon` is as following:
...
-            default: 1
+        -c, --cudapoa-batches
+            default: 0
             number of batches for CUDA accelerated polishing
+        -b, --cuda-banded-alignment
+            default: 0
+            use banding approximation for alignment on GPU
+        --cudaaligner-batches (experimental)
+            default: 0
+            Number of batches for CUDA accelerated alignment
```
Comment (vellamike) [inline README.md]: We should clarify that 0 means don't use GPU

File: README.md [hunk 2]
```diff
@@ -138,9 +108,15 @@ Usage of `racon` is as following:
...
 
     only available when built with CUDA:
-        -c, --cuda-batches
-            default: 1
+        -c, --cudapoa-batches
+            default: 0
             number of batches for CUDA accelerated polishing
+        -b, --cuda-banded-alignment
+            default: 0
+            use banding approximation for alignment on GPU
```
Comment (vellamike) [inline README.md]: What is the behaviour if -c=0 and -b !=0? This should be clarified

File: README.md [hunk 3]
```diff
@@ -138,9 +108,15 @@ Usage of `racon` is as following:
             prints the usage
 
     only available when built with CUDA:
-        -c, --cuda-batches
-            default: 1
+        -c, --cudapoa-batches
+            default: 0
```
Comment (vellamike) [inline README.md]: We should clarify that 0 means don't use GPU

File: README.md (hunk 2 above)
Comment (vellamike) [inline README.md]: What is the behaviour if -c=0 and -b !=0? This should be clarified

File: README.md [hunk 4]
```diff
@@ -138,9 +108,15 @@ Usage of `racon` is as following:
...
             number of batches for CUDA accelerated polishing
+        -b, --cuda-banded-alignment
+            default: 0
+            use banding approximation for alignment on GPU
+        --cudaaligner-batches (experimental)
+            default: 0
```
Comment (vellamike) [inline README.md]: We should clarify that 0 means don't use GPU

File: README.md [hunk 5]
```diff
@@ -19,7 +19,12 @@ A **wrapper script** is also available to enable easier usage to the end-user fo
 
 ## Dependencies
 1. gcc 4.8+ or clang 3.4+
-2. cmake 3.2+ (cmake 3.10+ for CUDA support)
+2. cmake 3.2+
+
+### CUDA Support
+1. gcc 5.0+
+2. cmake 3.10+
+4. CUDA 9.0+
```
Comment (vellamike) [inline README.md]: Is this right? I thought we run our CI on CUDA10?

Comment (tijyojwad) [reply README.md]: The Ubuntu CI job runs CUDA 9.2, whereas the CentOS job runs CUDA 10. I'm okay making it standard CUDA 10

File: README.md (hunk 3 above)
Comment (tijyojwad) [reply README.md]: This was a mistake. The default actually is 1. So if someone doesn't specify `-c X` and only uses `-c` then only a single batch is used. If `-c` is not specified then CUDA polishing is not used

File: README.md (hunk 2 above)
Comment (tijyojwad) [reply README.md]: This shouldn't need a default value either. `-b` just means that banding is used, and not if `-b` is not specified. Clarified that in the doc

File: README.md (hunk 4 above)
Comment (tijyojwad) [reply README.md]: This option needs an argument for now, so no default values

File: README.md (hunk 5 above)
Comment (vellamike) [reply README.md]: OK let's change to 10

**Pull request merged**